### PR TITLE
Rewrite Discord group sync to use v5 Nameless-Link API

### PR DESCRIPTION
**Pull request by tadhgboyle**

### CODE REVIEW

File: modules/Core/includes/endpoints/InfoEndpoint.php
Comment (tadhgboyle) [inline modules/Core/includes/endpoints/InfoEndpoint.php]: Undo before merge

**Review by Derkades — DISMISSED** [review]
You've requested my review but I don't know enough about NamelessMC to review this code.

I will say I am happy with the current API, from the perspective of Nameless-Plugin and Nameless-Link. And you tested it and it worked, so that's a good sign (although all previous group sync system iterations also appeared to work)

You will need a second approval anyway, after undoing `2.1.0` line :P

**Review by supercrafter100 — DISMISSED** [review]

**Review by Derkades — APPROVED** [review]

**Pull request merged**
